### PR TITLE
feat: add error text on lacking text

### PR DESCRIPTION
**Pull request by magsyg**
Seems textitems would just show blank text if text item did not exist, instead just added error message for easier error finding

### CODE REVIEW

File: frontend/src/hooks.ts [hunk 1]
```diff
@@ -75,9 +76,19 @@ export function useTextItem(key: TextItemValue, language?: string): string | und
...
-    });
+    getTextItem(key)
+      .then((data) => {
+        setTextItem(data);
+      })
+      .catch((error) => {
+        if (error.request.status === STATUS.HTTP_404_NOT_FOUND) {
+        }
+        setTextItem({
+          key: 'HEST',
```
Comment (Snorre98) [inline frontend/src/hooks.ts:87]:
```suggestion
          key: 'MISSING_TEXTITEM',
```

File: frontend/src/hooks.ts [hunk 2]
```diff
@@ -75,9 +76,19 @@ export function useTextItem(key: TextItemValue, language?: string): string | und
...
+          text_nb: 'ERROR ERROR kontakt MG:WEB om manglende tekst ERROR ERROR',
```
Comment (Snorre98) [inline frontend/src/hooks.ts:88]:
```suggestion
          text_nb: 'Mangler tekst innhold. Kontakt redaksjon@samfundet.no',
```

File: frontend/src/hooks.ts [hunk 3]
```diff
@@ -75,9 +76,19 @@ export function useTextItem(key: TextItemValue, language?: string): string | und
...
+          text_en: 'ERROR ERROR contact MG:WEB about lacking text ERROR ERROR',
```
Comment (Snorre98) [inline frontend/src/hooks.ts:89]:
```suggestion
          text_en: 'Missing text content. Contact redaksjon@samfundet.no',
```

**Review by Snorre98 — APPROVED** [review]
Se kommentarer

File: frontend/src/hooks.ts (hunk 3 above)
Comment (magsyg) [reply frontend/src/hooks.ts]: Done

File: frontend/src/hooks.ts (hunk 2 above)
Comment (magsyg) [reply frontend/src/hooks.ts]: Done

File: frontend/src/hooks.ts (hunk 1 above)
Comment (magsyg) [reply frontend/src/hooks.ts]: Done

**Pull request merged**